caampr-backend/src/main/java/com/jelistan/caampr/lambda/provider/GearProvider.java

```diff
@@ -1,16 +1,15 @@
 package com.jelistan.caampr.lambda.provider;
 
-import com.google.common.collect.Lists;
 import com.jelistan.caampr.lambda.model.Gear;
 import com.jelistan.caampr.lambda.model.Link;
 import com.jelistan.caampr.lambda.model.LinkTypes;
 
 import java.util.List;
 
-public class GearProvider {
+public interface GearProvider {
 
     public static final Gear FAKE_GEAR = Gear.builder()
```

Comment:
a constant defined in an interface does not require the `public static final` part, it's redundant

Reply:
removed final